src/item/BoatType.php

```diff
@@ -54,12 +55,14 @@ protected static function setup() : void{
 			new self("acacia", WoodType::ACACIA()),
 			new self("dark_oak", WoodType::DARK_OAK()),
 			new self("mangrove", WoodType::MANGROVE()),
+			new self("bamboo", WoodType::BAMBOO(), "Raft"),
 		);
 	}
 
 	private function __construct(
 		string $enumName,
 		private WoodType $woodType,
+		private readonly ?string $suffix = null,
```

Comment:
Rather than setting it to null here, wouldn't it be better to set ``boat``. This would avoid all conditions later

Reply:
I follow the api style of WoodType enum, using a hard code string can be problematic